app/styles/partials/utils.scss

```diff
@@ -7,3 +7,6 @@
 .scrollable-modal {
   max-height: 50vh;
 }
+.l-margin {
```

Comment:
@harshitagupta Instead of naming it `l-margin` please rename it to something specific to the attendees label or some other appropriate name, l-margin is mis leading in terms of what this class is doing.